<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecCmd</class>
 <widget class="QDockWidget" name="ExecCmd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exec Cmd</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="CmdLabel">
      <property name="text">
       <string>Script</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLineEdit" name="ParamEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="ParamLabel">
      <property name="text">
       <string>Parameters</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="OutLabel">
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLineEdit" name="ScriptEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="CmdBox"/>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="ExecBtn">
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="OutText"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="ProgLabel">
      <property name="text">
       <string>Program</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>269</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QToolButton" name="ScriptTool">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
